docs/resources/azure_vpn_gateways.md

```diff
@@ -0,0 +1,85 @@
+---
+title: About the azure_vpn_gateways Resource
+platform: azure
+---
+
+# azure_vpn_gateways
+
+Use the `azure_vpn_gateways` InSpec audit resource to test properties and configuration of multiple Azure VPN gateways.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_vpn_gateways` resource block returns all VPN gateways, either within a Resource Group (if provided), or within an entire Subscription.
```

Comment:
Seems like the `resource_group` is a mandatory URI parameter, [here](https://docs.microsoft.com/en-us/rest/api/network-gateway/virtualnetworkgateways/list#uri-parameters).

Reply:
Hi Omer, I've renamed that now in my latest commit.

Reply:
Upon testing this it doesn't seem to be mandatory, I can call it with and without a resource group and it seems to behave as expected.

Reply:
Perfect, thanks for the clarification!